client/js/components/assessments/item.jsx

```diff
@@ -24,16 +24,10 @@ export default class Item extends React.Component{
     // or ungraded. Should match response constants declared below.
     messageIndex      : React.PropTypes.number.isRequired,
 
-    goToNextQuestion  : React.PropTypes.func.isRequired,
-
-    goToPrevQuestion  : React.PropTypes.func.isRequired,
-
-    submitAssessment  : React.PropTypes.func.isRequired,
-
     selectAnswer      : React.PropTypes.func.isRequired,
 
     // TODO
-    outcomes          : React.PropTypes.object,
+    outcomes          : React.PropTypes.array,
```

Comment:
Outcomes selector was giving us an array. I changed this to array to get rid of warning message. I don't believe we are actually doing anything with outcomes just yet.